### ISSUE TITLE
[Feature Request / Feature Improvement] Policies autocomplete

### ISSUE THREAD

**Issue by filhocodes** [laravel-idea/plugin#210]
Right now, when the second parameter of Gate functions is filled, it only suggests completions from Model Policies when the parameter is an Eloquent Model. However, there is a possibility of the second parameter being an array.

```php

// Suggest completions for model:
$this->authorize('policy', $model)

// Doesn't suggest completions:
$this->authorize('policy', [$model])
$this->authorize('policy', [$model, $extraParameter])

 ```
 
 The request to improve this feature: **If the parameter for the Gate function is an array, and the first element is an Eloquent Model, suggest policies for the associated Policy Model**.

**Comment by adelf** [comment]
Thanks!

**Comment by adelf** [comment]
Fixed by 4.1.1 released today.
